noweekend-mcp/mcp-host/src/main/kotlin/noweekend/mcphost/controller/ChatbotController.kt

```
@@ -38,6 +39,10 @@ class ChatbotController(
         return ResponseEntity.ok(result)
     }
 
+    @GetMapping("/getHoliday")
+    fun getHolidays() {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)